search-service/src/main/kotlin/com/egm/stellio/search/csr/service/ContextSourceCaller.kt

```diff
@@ -52,19 +109,19 @@ object ContextSourceCaller {
             }
             .header(HttpHeaders.LINK, httpHeaders.getFirst(HttpHeaders.LINK))
```

Comment:
won't it complain if `httpHeaders.getFirst(HttpHeaders.LINK)` is null?

Reply:
I think I've seen a test where the mock does not return a Link header and it has not raised any issue

Reply:
The previous implementation was sending a link header with no value. Which work in the test but would probably break when calling another broker.